schemas/IDLs/org.gel.models.participant.avro/1.2.0/CommonParticipant.avdl

```diff
@@ -167,7 +167,7 @@ protocol CommonParticipant {
     /**
     Consent Status
     */
-    record ConsentStatus {
+    record GmsConsentStatus {
```

Comment:
Just to avoid confusion, is not NGIS == GMS? If so I would stick to NGIS as we have used it elsewhere.

Reply:
@priesgo in fact we should change everything to Gms.  NGIS is the project to initiate the GMS